test/failover/failover.test.lua

```diff
@@ -177,6 +177,7 @@ cfg.failover_ping_timeout = 0.0000001
 vshard.router.cfg(cfg)
 while not test_run:grep_log('router_1', 'Ping error from', 1000) do fiber.sleep(0.01) end
 
+collectgarbage("stop")
```

Comment:
I suggest to write a comment here why is it needed. Also you might want to try to save the netbox connection into a variable. Then you would keep one strong reference and prevent the GC + you wouldn't have to pause the entire GC for everything else.

Reply:
Added the comment, ref is ok too, it also fixes the flakiness. So the problem is definitely in Lua GC.